config/_default/config.toml

```diff
@@ -0,0 +1,29 @@
+baseURL = "https://www.pulumi.com/"
+languageCode = "en-us"
+title = "Pulumi"
+
+disableKinds = ["category", "taxonomyTerm"]
+sectionPagesMenu = "main"
+pygmentsCodeFences = true
+pygmentsUseClasses = true
+
+# Don't crawl non-prod instances.
+enableRobotsTXT = false
```

Comment:
This has the effect of not _generating_ a robots.txt, which effectively tells the bots it's OK to be indexed. We should enable robots.txt in all environments, and then unless the Hugo env is `production`, `Disallow: *`.

Reply:
Ah, good catch.